src/Shared/Http2cat/HPackHeaderWriter.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.HPack;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2
+{
+    internal static class HPackHeaderWriter
```

Comment:
This is copy and paste (with some modifications to remove some internal type dependencies) but that appears to match the other code in http2cat which is copied from Http2TestBase